lib/absinthe/resolution.ex

```diff
@@ -128,6 +128,7 @@ defmodule Absinthe.Resolution do
   In this case `5` is the 0 based index in the list of users the field is currently
   at.
   """
+  @spec path(t()) :: [String.t()]
   def path(%{path: path}) do
```

Comment:
I don't think this spec is correct. This returns the query nodes, not strings. Also, in the case of lists, it returns integers.

Reply:
I'll switch to `[String.t() | integer()]`